### PR TITLE
Adding usernotes

### PR DESCRIPTION
**Pull request by mjhouse**
This adds a dialog accessible only to mods by clicking '\_U\_' next to a users name on the comments/posts and post listing pages. The dialog allows mods to create usernotes tagged with the labels listed in #14.
Screenshot from 2022-05-14 11-59-19

### CODE REVIEW

File: files/helpers/wrappers.py [hunk 1]
```diff
@@ -22,16 +22,18 @@ def get_logged_in_user():
...
 			if v:
 				nonce = session.get("login_nonce", 0)
-				if nonce < v.login_nonce or v.id != id: abort(401)
+				if nonce < v.login_nonce or v.id != id:
+					abort(401)
 
 				if request.method != "GET":
 					submitted_key = request.values.get("formkey")
-					if not submitted_key: abort(401)
-					if not v.validate_formkey(submitted_key): abort(401)
```
Comment (zorbathut) [inline files/helpers/wrappers.py]: It looks like the changes in this file aren't actually changes, just a little reformatting. They seem like reasonable reformatting to do but can we split this into a separate change request?

File: files/templates/comments.html [hunk 2]
```diff
@@ -1,916 +1,925 @@
-{% if not ajax %}
```
Comment (zorbathut) [inline files/templates/comments.html]: Github is flagging this entire file as changed :V I think it must've been an indent or CRLF change; can this be simplified? Alternatively, if it really is a necessary formatting change, gimme a pull request for the formatting change *first* to make the real content change less intrusive and easier to track later.

**Review by zorbathut — CHANGES_REQUESTED** [review]

**Comment by zorbathut** [comment]
Overall: Looks really good! I've got two comments and both of them are just cleaning up the changelist a little.

If we've got the Alembic stuff in by the time this is ready then I'll also be asking you to do the Alembic update process, otherwise we'll change the Alembic changelist to take this into account instead :)

File: files/helpers/wrappers.py (hunk 1 above)
Comment (mjhouse) [reply files/helpers/wrappers.py]: That was left over from debug code I added and removed while I was figuring out how to send POST requests to the api. I've reverted it.

File: files/templates/comments.html (hunk 2 above)
Comment (mjhouse) [reply files/templates/comments.html]: I'm not sure why it was saying the whole file was changed, but I rolled it back, reintroduced my changes and up-merged from master and it isn't flagging it anymore.

**Comment by mjhouse** [comment]
@zorbathut I think I've resolved your issues, but you have to do the actual merge. Let me know if you want me to do anything else.

**Review by zorbathut — APPROVED** [review]

**Pull request merged**